rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientRegisterActivityTest.java

```diff
@@ -44,6 +45,19 @@ public void setUp() {
                 .get();
     }
 
+    @Test
+    public void testVerifyGlobalActivityShouldBeCovidPatientRegisterActivity() throws Exception {
+        RDTApplication.getInstance().clearCurrActivityReference(covidPatientRegisterActivity);
+        Whitebox.invokeMethod(covidPatientRegisterActivity, "onResume");
+        Assert.assertThat(RDTApplication.getInstance().getCurrentActivity(), CoreMatchers.instanceOf(CovidPatientRegisterActivity.class));
+    }
+
+    @Test
+    public void testVerifyGlobalActivityShouldBeNull() throws Exception {
```

Comment:
rename to `testOnPauseShouldClearCurrentActivity`

Reply:
done